routes/nodebb.js

```diff
@@ -125,6 +125,7 @@ app.delete(`${BASE_REPORT_URL}/v2/groups/:slug/membership/:uid`, proxyObject());
 
 // post apis 
 app.get(`${BASE_REPORT_URL}/post/pid/:pid`, proxyObjectWithoutAuth());
+app.get(`${BASE_REPORT_URL}/v3/posts/:pid`, proxyObjectWithoutAuth());
```

Comment:
Add line comments saying this is used for v1.18.x

also add line comment for previous API saying need to deprecated in the next upgrade. Used for v.1.16.x